js-api-doc/compile.d.ts

```diff
@@ -44,8 +44,9 @@ export interface CompileResult {
  * @category Compile
  */
 export class Compiler {
-  /**Throws an error if constructed directly, instead of via
-   * {@link initCompiler}. */
+  /** Throws an error if constructed directly, instead of via
```

Comment:
Standard JSDoc style (as used by other comments here) has the first `/**` on its own line.

Reply:
Fixed in https://github.com/sass/sass/pull/3744/commits/11f1af3748befbd7e21e7a6e5cce55caf2b4ff8a